Advanced_Projects/PDFTextGenie/ocr/llama_ocr.py

```
@@ -0,0 +1,66 @@
+import easyocr
+import numpy as np
+
+# TrOCR imports
+from transformers import TrOCRProcessor, VisionEncoderDecoderModel
+import torch
+from PIL import Image
+
+class EasyOCREngine:
+    def __init__(self, languages=['en'], gpu=False):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)